tests/build/build.go

```diff
@@ -505,16 +505,26 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 				for i := range gatherResult {
 					if gatherResult[i] == "sanity-inspect-image" {
+						// Fetching HACBS_TEST_OUTPUT shouldn't fail
 						result, err := build.FetchImageTaskRunResult(pipelineRun, gatherResult[i], "BASE_IMAGE")
 						Expect(err).ShouldNot(HaveOccurred())
 						ret := build.ValidateImageTaskRunResults(gatherResult[i], result)
 						Expect(ret).Should(BeTrue())
-						// TODO conftest-clair returns SUCCESS which is not expected
-						// } else {
-						// 	result, err := build.FetchTaskRunResult(pipelineRun, gatherResult[i], "HACBS_TEST_OUTPUT")
-						// 	Expect(err).ShouldNot(HaveOccurred())
-						// 	ret := build.ValidateTaskRunResults(gatherResult[i], result)
-						// 	Expect(ret).Should(BeTrue())
+					}
+					if gatherResult[i] == "conftest-clair" {
```

Comment:
should it be changed to
```suggestion
					} else if gatherResult[i] == "conftest-clair" {
```
?